spynnaker/pyNN/models/neural_projections/connectors/abstract_connector.py

```diff
@@ -132,7 +122,7 @@ def _get_delay_maximum(self, delays, n_connections):
             type(delays)))
 
     @abstractmethod
-    def get_delay_maximum(self, delays):
+    def get_delay_maximum(self, delays, synapse_info):
```

Comment:
A question here before I head off and do this: these parameter are called "weights" and "delays" everywhere except in SynapseInformation.  Should I change it to this in here too, especially since they're going to be coming from there directly now?

Reply:
Hmm, yes I guess that makes sense here.  The synapse types in PyNN (e.g. StaticSynapse) use weight and delay without the plural, but I am happy for our code to use the plural.